ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -280,6 +280,24 @@
    */
   CompletableFuture<ConnectorDescription> describeConnector(String connectorName);
 
+  /**
+   * Define a session variable which can be referenced in sql commands by wrapping it with ${}
+   *
```

Comment:
Clarify the behavior when the variable already exists (in this javadoc). The current behavior is that the value will be overridden. Is that the canonical choice, rather than throwing an error? Would be good to have someone from the product team sanity check these new interfaces and their expected behaviors.

Similarly, undefine currently does not throw an error if the variable name does not exist. (Clarify in javadoc and sanity check expected behavior.)